### PR TITLE
SUP-117-Transform-Data-Audit

### PR DESCRIPTION
**Pull request by edgargarciamacrometa**
In the current documentation, there is an explanation of the input stream TempStream, but we are missing the OutputStream. The schema for OutputStream is roomNo and avgTemp, the values received from TempStream are processed by the query and inserted into the output stream. 

Also source stream was changes to SINK stream

### CODE REVIEW

File: docs/cep/tutorials/transforming-data.md [hunk 1]
```diff
@@ -22,70 +22,61 @@ the temperature of a room every second are transformed to indicate the average t
 5. Add the following sample stream application.
```
Comment (diana-macrometa) [inline docs/cep/tutorials/transforming-data.md]: In all cases, change "stream application" to "stream worker."

They used to call stream workers "stream applications." This is now outdated terminology, like calling Macrometa C8DB.

File: docs/cep/tutorials/transforming-data.md [hunk 2]
```diff
@@ -22,70 +22,61 @@ the temperature of a room every second are transformed to indicate the average t
...
 6. Let's define the input stream to define the schema based on which data is selected to the streaming integration flow.
```
Comment (diana-macrometa) [inline docs/cep/tutorials/transforming-data.md]:
```suggestion
6. Let's define the input stream to define the schema based on which data are selected to the streaming integration flow.
```

File: docs/cep/tutorials/transforming-data.md [hunk 3]
```diff
@@ -22,70 +22,61 @@ the temperature of a room every second are transformed to indicate the average t
...
 
-    In this example, let's assume that each event indicates the device ID, the room ID, and the temperature. Therefore, let's define an input stream as follows:
-
+    In this example, let's assume that each event indicates the device ID, the room number, and the temperature. Therefore, let's define an input stream as follows:
```
Comment (diana-macrometa) [inline docs/cep/tutorials/transforming-data.md]: Reading through this set of instructions, it seems like almost every use of "let's" is unnecessary. Try deleting them and see how it reads.

File: docs/cep/tutorials/transforming-data.md [hunk 4]
```diff
@@ -22,70 +22,61 @@ the temperature of a room every second are transformed to indicate the average t
...
            
-          
-4. To do the required transformation, let's add the query as follows:
+7. Let's define the output stream which will receive the average temperature of each incomming message in the TempStream. The output stream definition is as follows:
+    ```sql
+       CREATE SINK OutputStream WITH (type='stream', stream='OutputStream', map.type='json') (roomNo int, avgTemp double);
+    ```
 
+8. To do the required transformation, let's add the query as follows:
+    1. Add the `INSERT INTO` clause with the name of the output stream to indicate that the processed events are directed to that stream.
```
Comment (diana-macrometa) [inline docs/cep/tutorials/transforming-data.md]: Do not use numbers as a sublist, use letters.

File: docs/cep/tutorials/transforming-data.md [hunk 5]
```diff
@@ -22,70 +22,61 @@ the temperature of a room every second are transformed to indicate the average t
...
+    
+    CREATE SINK OutputStream WITH (type='stream', stream='OutputStream', map.type='json') (roomNo int, avgTemp double);
+    
+    INSERT INTO OutputStream
+    SELECT roomNo, avg(temp) AS avgTemp
+    FROM TempStream
+    GROUP BY roomNo;
     ```
 
 ## Transform data using Plugins
```
Comment (diana-macrometa) [inline docs/cep/tutorials/transforming-data.md]:
```suggestion
## Transform Data Using Plugins
```

**Review by diana-macrometa — CHANGES_REQUESTED** [review]

**Comment by diana-macrometa** [comment]
When this is ready for re-review, request one by clicking the circling arrows next to my name.

File: docs/cep/tutorials/transforming-data.md [hunk 6]
```diff
@@ -6,89 +6,81 @@ sidebar_position: 4
 
 ## Introduction
 
-The stream processor allows you to perform a wide range of transformations to the input data received. The transformations are carried out via operators that are defined inline within the stream application.
+The stream processor allows you to perform a wide range of transformations to the input data received. The transformations are carried out via operators that are defined inline within the stream worker.
 
 ## Transform data using Operators
 
-The operators that you can configure inline within Stream applications in order to carry out data transformations are listed in the [Stream Query Guide](../query-guide/index.md).
+The operators that you can configure inline within Stream workers in order to carry out data transformations are listed in the [Stream Query Guide](../query-guide/index.md).
```
Comment (diana-macrometa) [inline docs/cep/tutorials/transforming-data.md]:
```suggestion
The operators that you can configure inline within stream workers in order to carry out data transformations are listed in the [Stream Query Guide](../query-guide/index.md).
```

File: docs/cep/tutorials/transforming-data.md [hunk 7]
```diff
@@ -6,89 +6,81 @@ sidebar_position: 4
...
 
 To show how an inline operators are configured, let's consider an example where readings from a sensor that indicates 
 the temperature of a room every second are transformed to indicate the average temperature and the average humidity as at each second.
 
 1. Open the GUI. Click the **Stream Apps** tab.
```
Comment (diana-macrometa) [inline docs/cep/tutorials/transforming-data.md]:
```suggestion
1. Log in to the Macrometa web console. Click the **Stream Apps** tab.
```

File: docs/cep/tutorials/transforming-data.md [hunk 8]
```diff
@@ -6,89 +6,81 @@ sidebar_position: 4
...
+    INSERT INTO OutputStream
+    SELECT roomNo, avg(temp) AS avgTemp
+    FROM TempStream
+    GROUP BY roomNo;
     ```
 
-## Transform data using Plugins
+## Transform Data Using Plugins
 
 The stream processor offers a variety of options to carry out data transformations via in-built plugins. The following table describes the complete list of plugins that provide data transformation functionality.
```
Comment (diana-macrometa) [inline docs/cep/tutorials/transforming-data.md:85]: By stream processor, do you mean stream worker?

Comment (edgargarciamacrometa) [reply docs/cep/tutorials/transforming-data.md:85]: I think stream processor, 

Stream Processor

This allows events to be collected, generated, dropped and expired anytime by altering the event format by adding one or more attributes to it based on the given input parameters.

**Review by diana-macrometa — APPROVED** [review]
Looks good!

**Pull request merged**